tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -148,7 +148,7 @@ class WskBasicUsageTests
         }
     }
 
-    it should "show api build using http apihost" in {
+    it should "show api build using http apihost with protocol specified" in {
```

Comment:
@lzbj - is another successful test needed that provides the api host without the protocol being provided?